tests/unit/lms/services/vitalsource_test.py

```diff
@@ -58,10 +58,11 @@ def test_it_signs_lti_launch_form_params(self, svc, lti_user):
         # Ignore non-OAuth signature params in this test.
         params = {k: v for (k, v) in params.items() if k.startswith("oauth_")}
 
+        base64_char = "[0-9a-zA-Z+=/]"
         assert params == {
             "oauth_consumer_key": "lti_launch_key",
             "oauth_nonce": Any.string.matching("[0-9]+"),
-            "oauth_signature": Any.string.matching("[0-9a-zA-Z+=]+"),
+            "oauth_signature": Any.string.matching(f"{base64_char}+"),
```

Comment:
This check doesn't fully validate that the string is valid base 64, but I deemed it good enough for the purposes of this test.